packages/manager/src/factories/cloudpulse/alerts.ts

```diff
@@ -29,6 +32,42 @@ export const alertRulesFactory = Factory.Sync.makeFactory<AlertDefinitionMetricC
 );
 
 const alertTypes: AlertDefinitionType[] = ['system', 'user'];
+export const triggerConditionFactory = Factory.Sync.makeFactory<TriggerCondition>(
+  {
+    criteria_condition: 'ALL',
+    evaluation_period_seconds: 300,
+    polling_interval_seconds: 60,
+    trigger_occurrences: 3,
+  }
+);
+export const rulesFactory = Factory.Sync.makeFactory<MetricCriteria>({
+  aggregate_function: 'avg',
+  dimension_filters: [
+    {
+      dimension_label: 'region',
+      operator: 'eq',
+      value: 'us-ord',
+    },
+  ],
+  metric: 'cpu_usage',
+  operator: 'gte',
+  threshold: 1000,
+});
```

Comment:
```suggestion
  aggregate_function: 'avg',
  dimension_filters: [
    {
      dimension_label: 'state',
      operator: 'eq',
      value: 'idle',
    },
  ],
  metric: 'system_cpu_utilization_percent',
  operator: 'gte',
  threshold: 1000,
});
```
These values in the edit flow are dependent on the `metric-definitions` query. So if the values do not match the values from the mocks of metric-definitions. The default values in edit form won't function ideally.